Intersect.Client/Intersect.Client.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <TargetFramework>net6.0-windows</TargetFramework>
```

Comment:
It does break it, and was only required for the clipboard. I'll be investigating how to support the clipboard without it.

Reply:
Yeah it seems that the Clipboard class is not supported anymore with .net 6.0.
It seems that it was supported up to .NET Framework 4.8 and then it moved to WPF somehow.

I found a possible solution with platform-support. It's TextCopy (also available as nuget package): https://github.com/CopyText/TextCopy
Would be also license compatible since it is MIT.

Found it from this stackoverflow thread: https://stackoverflow.com/questions/44205260/net-core-copy-to-clipboard

I never did something with that library but it looks promising. Downside is: we would have a dependency more.